v0/destinations/indicative/transform.js

```diff
@@ -7,9 +7,71 @@ const {
   defaultPostRequestConfig
 } = require("../../util");
 
+const handleProperties = properties => {
+  const result = {};
+  let l;
+
+  const recurse = (cur, prop) => {
+    let i;
+    if (Object(cur) !== cur) {
+      // primitive types
+      result[prop] = cur;
+    } else if (Array.isArray(cur)) {
+      if (cur.length > 0) {
+        const fel = cur[0];
+        if (Object(fel) !== fel) {
+          result[prop] = cur.toString();
+        } else if (Array.isArray(fel)) {
+          // ignore array or arrys
+        } else {
+          const objectMap = {};
+          Object.keys(fel).forEach(key => {
+            // discarding nested objects and arrays for array of objects
+            // only allowing primitive times for array of objects
+            if (Object(fel[key]) !== fel[key]) {
```

Comment:
!==. ==> == 
wrong comparison?

Reply:
Only allowing the non-object and non-array types